poem/Poem/poem/admin_interface/siteprobes.py

```diff
@@ -238,8 +244,27 @@ def save_model(self, request, obj, form, change):
         if request.user.has_perm('poem.groupown_probe') \
                 or request.user.is_superuser:
             obj.user = request.user.username
-            obj.save()
-            return
+            if form.cleaned_data['new_version'] and change == True or change == False:
+                obj.save()
+                return
+            elif not form.cleaned_data['new_version'] and change == True:
+                version = Version.objects.get_for_object(obj)
+                pk = version[0].object_id
+                pk0 = version[0].id
+                data = json.loads(Version.objects.get(pk=pk0).serialized_data)
+                newfield = dict()
+                newfield['name'] = form.cleaned_data['name']
+                newfield['version'] = form.cleaned_data['version']
+                newfield['description'] = form.cleaned_data['description']
+                newfield['comment'] = form.cleaned_data['comment']
+                newfield['repository'] = form.cleaned_data['repository']
+                newfield['docurl'] = form.cleaned_data['docurl']
+                newfield['group'] = obj.group
+                newfield['user'] = obj.user
+                newdata = data
+                newdata[0]['fields'] = newfield
+                Version.objects.filter(pk=version[0].object_id).update(serialized_data=json.dumps(newdata))
```

Comment:
This is a error. You're interested in primary key from `reversion_version` table. So:
```
Version.objects.filter(pk=pk0).update(serialized_data=json.dumps(data))
```